sdks/python/apache_beam/runners/dataflow/dataflow_runner_test.py

```diff
@@ -323,7 +316,9 @@ def test_no_group_by_key_directly_after_bigquery(self):
                          options=PipelineOptions(self.default_properties)) as p:
         # pylint: disable=expression-not-assigned
         p | beam.io.Read(
-            beam.io.BigQuerySource('dataset.faketable')) | beam.GroupByKey()
+            beam.io.BigQuerySource(
+                'dataset.faketable',
+                use_dataflow_native_source=True)) | beam.GroupByKey()
```

Comment:
I assume the state is following.
(1) We have full test coverage for the new source (which is the default)
(2) We have additional tests that check old native source which can be just removed whenever we discontinue support for that source.

Reply:
for these two tests in dataflow_runner_test, we do not have equivalents for the custom source. This is because these failures do not occur under the custom source:
- Custom source is acceptable in streaming because it becomes an SDF that runs fine in streaming
- Custom source will not error out on type hints with GBK because the type hints are not set up to fail for this sort of composite transform. I don't know why, but we can file a bug to ensure this is re-enabled.